common/workunit/wuattr.cpp

```diff
@@ -283,3 +305,16 @@ void setAttributeValueInt(IPropertyTree & tgt, WuAttr kind, __int64 value)
     else
         addGraphAttribute(&tgt, info.childPath)->setPropInt64("@value", value);
 }
+
+bool isListAttribute(WuAttr kind)
+{
+    const WuAttrInfo & info = attrInfo[kind-WaNone];
+    return (info.multiKind && info.multiKind == kind);
```

Comment:
shouldn't the first part of this condition be info.multiKind != WaNone?  All non-list  AttrInfo are initialized with WaNone as the multiKind and WaNone has value > 0

Reply:
well spotted.